src/pages/card.js

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import "./card.css";
+import { SlLogin } from "react-icons/sl";
+
+function Card() {
+  const content = 150;
```

Comment:
In `React`, to make it configurable, we need to pass them as `props`. Please change all configurable elements of the component as `props` instead and use the passed value in your component.

You can find out more on how to do this here. https://react.dev/learn/passing-props-to-a-component